archlinuxcn/pid_children-git/tmpfiles.conf

```diff
@@ -0,0 +1 @@
+z /usr/bin/kill_children - - pkg
```

Comment:
Shouldn't we specify permissions here? Changing group would discard the sgid bit.

Reply:
Thanks I forgot that is how chown/chgrp usually works. But again systemd does something different - it tries to keep suid/sgid as much as possible (https://github.com/systemd/systemd/blob/v244/src/tmpfiles/tmpfiles.c#L833). Anyway specifying permissions explicitly is better.

Reply:
What? It means that I can get root for free if I can find a systemd-managed root owned file that I can unlink and recreate....